src/screens/SearchScreen.spec.js

```diff
@@ -3,24 +3,20 @@ import { __esModule } from "@testing-library/jest-dom/dist/matchers";
 import { render, waitFor, screen } from "@testing-library/react";
 import SearchScreen from "./SearchScreen";
 import "@testing-library/jest-dom";
-import { BrowserRouter } from "react-router-dom";
-import { LoadScript } from "@react-google-maps/api";
 
 jest.mock("react-router-dom", () => {
   const originalModule = jest.requireActual("react-router-dom");
   return {
-    __esModule: true,
```

Comment:
Removed __esModule since its not needed here and testcases run perfectly without this config, hence jest.config.js not added.